### PR TITLE
Feat/room service room and offer

### PR DESCRIPTION
**Pull request by mailo001**

### CODE REVIEW

File: server/pkg/postgres/room_storage.go [hunk 1]
```diff
@@ -235,8 +251,8 @@ func (r *RoomStorage) OffersRelatedWithRoom(ctx context.Context, roomID int64) (
 	return result, nil
 }
 
-// RoomsRelatedWithRoom implements database feat to get rooms IDs related with offer
-func (r *RoomStorage) RoomsRelatedWithRoom(ctx context.Context, offerID int64) ([]int64, error) {
+// RoomsRelatedWithOffer implements database feat to get rooms IDs related with offer
+func (r *RoomStorage) RoomsRelatedWithOffer(ctx context.Context, offerID int64) ([]int64, error) {
```
Comment (Gloomwidow) [inline server/pkg/postgres/room_storage.go:255]: Maybe "GetRoomsRelatedToOffer"?

File: server/pkg/postgres/room_storage.go [hunk 2]
```diff
@@ -204,6 +199,27 @@ func (r *RoomStorage) GetRoom(ctx context.Context, roomID int64) (bookly.Room, e
 	return room, nil
 }
 
+// IsRoomBelongToHotel implements database feat to get room by ID
+func (r *RoomStorage) IsRoomBelongToHotel(ctx context.Context, roomID int64, hotelID int64) (bool, error) {
```
Comment (Gloomwidow) [inline server/pkg/postgres/room_storage.go:203]: Maybe `IsRoomOwnedByHotel` and `errRoomNotOwnedByHotel`?

It should be easier to read and understand.

File: server/pkg/postgres/room_storage.go (hunk 1 above)
Comment (mailo001) [reply server/pkg/postgres/room_storage.go]: Ok

File: server/pkg/postgres/room_storage.go (hunk 2 above)
Comment (mailo001) [reply server/pkg/postgres/room_storage.go]: Ok

**Review by Gloomwidow — APPROVED** [review]

**Pull request merged**